library/Zend/Http/Client.php

```diff
@@ -1041,7 +1041,7 @@ public function request($method = null)
                    if (! empty($query)) {
                        $query .= '&';
                    }
-                $query .= http_build_query($this->paramsGet, null, '&');
+                $query .= http_build_query($this->paramsGet);
```

Comment:
Should it be
```suggestion
                $query .= http_build_query($this->paramsGet, '', '&');
```
to guarantee the result

Reply:
thanks, fixed as you suggested